make testall fails because of cgroups_linux_test.go

## Version of Singularity:

Master on Ubuntu.

### Expected behavior

`make testall` should not fail.

### Actual behavior

```
--- FAIL: TestCgroups (0.01s)
    cgroups_linux_test.go:56: open /sys/fs/cgroup/blkio/singularity/12690/blkio.weight: permission denied
    cgroups_linux_test.go:63: can't determine cgroups root path, is cgroups enabled ?
panic: runtime error: invalid memory address or nil pointer dereference [recovered]
	panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x28 pc=0x8875c8]

goroutine 6 [running]:
testing.tRunner.func1(0xc0000cc200)
	/usr/local/go/src/testing/testing.go:830 +0x68a
panic(0x8e35c0, 0xc86c30)
	/usr/local/go/src/runtime/panic.go:522 +0x1b5
github.com/sylabs/singularity/internal/pkg/cgroups.(*Manager).Remove(0xc0000d7ed0, 0xc0000d7e80, 0x1)
	/home/gvallee/go/src/github.com/sylabs/singularity/internal/pkg/cgroups/cgroups_linux.go:133 +0x68
runtime.Goexit()
	/usr/local/go/src/runtime/panic.go:406 +0xed
testing.(*common).FailNow(0xc0000cc200)
	/usr/local/go/src/testing/testing.go:609 +0x5b
testing.(*common).Fatalf(0xc0000cc200, 0x94bfc6, 0x37, 0x0, 0x0, 0x0)
	/usr/local/go/src/testing/testing.go:672 +0x9f
github.com/sylabs/singularity/internal/pkg/cgroups.TestCgroups(0xc0000cc200)
	/home/gvallee/go/src/github.com/sylabs/singularity/internal/pkg/cgroups/cgroups_linux_test.go:63 +0xc70
testing.tRunner(0xc0000cc200, 0x94e8e8)
	/usr/local/go/src/testing/testing.go:865 +0x164
created by testing.(*T).Run
	/usr/local/go/src/testing/testing.go:916 +0x69a
FAIL	github.com/sylabs/singularity/internal/pkg/cgroups	0.026s
```

### Steps to reproduce behavior

`make testall`

--- a/internal/pkg/cgroups/cgroups_linux_test.go
+++ b/internal/pkg/cgroups/cgroups_linux_test.go
@@ -53,9 +53,8 @@ func TestCgroups(t *testing.T) {
 
 	manager := &Manager{Pid: pid, Path: path}
 	if err := manager.ApplyFromFile("example/cgroups.toml"); err != nil {
-		t.Errorf("%s", err)
+		t.Fatal(err)
 	}
-
 	defer manager.Remove()
 
 	rootPath := manager.GetCgroupRootPath()
@@ -132,8 +131,9 @@ func TestPauseResume(t *testing.T) {
 	manager.Path = filepath.Join("/singularity", strconv.Itoa(manager.Pid))
 
 	if err := manager.ApplyFromFile("example/cgroups.toml"); err != nil {
-		t.Errorf("%s", err)
+		t.Fatal(err)
 	}
+	defer manager.Remove()
 
 	manager.Pause()
 
@@ -181,8 +181,6 @@ func TestPauseResume(t *testing.T) {
 	}
 
 	file.Close()
-
-	defer manager.Remove()
 
 	pipe.Close()
 

